# STAT301 Final Project: Individual Assignment 1

**Group:** 26

**Group Member:** Sam Thorne 83910448

## Vancouver Crime Report Dataset:

 [Kaggle](https://www.kaggle.com/datasets/agilesifaka/vancouver-crime-report/data). 
 
 Talk about cutting it down and why we chose to cut it down. 
 Talk about where the data came from (where it was collected)
 

In [34]:
library(tidyverse)

In [36]:
data <- read_csv('https://raw.githubusercontent.com/samnthorne/FinalProject/main/data/crime_filtered.csv')
head(data)

Rows: 68545 Columns: 10
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (3): TYPE, HUNDRED_BLOCK, NEIGHBOURHOOD
dbl (7): YEAR, MONTH, DAY, HOUR, MINUTE, X, Y

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.


TYPE                       YEAR MONTH DAY HOUR MINUTE HUNDRED_BLOCK  
1 Break and Enter Commercial 2019  3     7   2    6     10XX SITKA SQ  
2 Break and Enter Commercial 2019  8    27   4   12     10XX ALBERNI ST
3 Break and Enter Commercial 2014  8     8   5   13     10XX ALBERNI ST
4 Break and Enter Commercial 2014  4    17   5   50     10XX ALBERNI ST
5 Break and Enter Commercial 2014  9     1  14   20     10XX ALBERNI ST
6 Break and Enter Commercial 2014 11    22  23   28     10XX ALBERNI ST
  NEIGHBOURHOOD X        Y      
1 Fairview      490613.0 5457110
2 West End      491007.8 5459174
3 West End      491015.9 5459166
4 West End      491032.3 5459150
5 West End      491032.3 5459150
6 West End      491059.5 5459122

## Descriptive summary:

Collecting information for descriptive summary of the data set:

In [52]:
print(summary(data))

     TYPE                YEAR          MONTH             DAY       
 Length:68545       Min.   :2014   Min.   : 1.000   Min.   : 1.00  
 Class :character   1st Qu.:2014   1st Qu.: 4.000   1st Qu.: 8.00  
 Mode  :character   Median :2019   Median : 6.000   Median :15.00  
                    Mean   :2017   Mean   : 6.318   Mean   :15.17  
                    3rd Qu.:2019   3rd Qu.: 9.000   3rd Qu.:22.00  
                    Max.   :2019   Max.   :12.000   Max.   :31.00  
                                                                   
      HOUR           MINUTE      HUNDRED_BLOCK      NEIGHBOURHOOD     
 Min.   : 0.00   Min.   : 0.00   Length:68545       Length:68545      
 1st Qu.: 7.00   1st Qu.: 0.00   Class :character   Class :character  
 Median :14.00   Median : 7.00   Mode  :character   Mode  :character  
 Mean   :12.38   Mean   :16.31                                        
 3rd Qu.:19.00   3rd Qu.:30.00                                        
 Max.   :23.00   Max.   :59.00

In [46]:
length((unique(data$TYPE)))
unique(data$TYPE)

[1] 25

[1] "Break and Enter Commercial"                            
 [2] "Break and Enter Residential/Other"                     
 [3] "Homicide"                                              
 [4] "Mischief"                                              
 [5] "Offence Against a Person"                              
 [6] "Other Theft"                                           
 [7] "Theft from Vehicle"                                    
 [8] "Theft of Bicycle"                                      
 [9] "Theft of Vehicle"                                      
[10] "Vehicle Collision or Pedestrian Struck (with Fatality)"
[11] "Vehicle Collision or Pedestrian Struck (with Injury)"

In [51]:
length((unique(data$HUNDRED_BLOCK)))
# unique(data$HUNDRED_BLOCK)

[1] 11275

In [49]:
length((unique(data$NEIGHBOURHOOD)))
unique(data$NEIGHBOURHOOD)

[1] 25

[1] "Fairview"                  "West End"                 
 [3] "Central Business District" "Grandview-Woodland"       
 [5] "Mount Pleasant"            "Strathcona"               
 [7] "Sunset"                    "Kensington-Cedar Cottage" 
 [9] "Stanley Park"              "Shaughnessy"              
[11] "Marpole"                   "West Point Grey"          
[13] "Hastings-Sunrise"          "Victoria-Fraserview"      
[15] "Kitsilano"                 "Kerrisdale"               
[17] "Riley Park"                "Oakridge"                 
[19] "Arbutus Ridge"             "Renfrew-Collingwood"      
[21] "Killarney"                 "Dunbar-Southlands"        
[23] "South Cambie"              "Musqueam"                 
[25] NA

The **Vancouver Crime DataSet** has 10 attributes to describe the 68545 crimes detailed in this dataset. Each row represents an instance of crime in Vancouver while each column represents details of these crimes. Of these 10 attributes, 7 of them are quantitative values (5 of which represent temporal data), while the remaining 3 are categorical attributes. 

**Vancouver Crime Data Attribute Summary**:

| Name of attribute | Type of data | Semantics | Notes |
| ---| ---| ---| ---|
| `TYPE`| Categorical `str`| Is a representation of the type of crime. |Has 25 unique values. This includes: `['Break and Enter Commercial','Break and Enter Residential/Other','Homicide','Mischief','Offence Against a Person','Other Theft''Theft from Vehicle','Theft of Bicycle','Theft of Vehicle','Vehicle Collision or Pedestrian Struck (with Fatality)','Vehicle Collision or Pedestrian Struck (with Injury)']` |
| `YEAR`| Categorical`dbl`| The year of the crime (2 options) | This is going to be either `2014` or `2019` because that is all we are including in our data analysis.|
|`MONTH`| Temporal Quantitative `dbl`| The month of the crime | Integars 1-12|
|`DAY`| Temporal Quantitative `dbl`| The day of the month of the crime | Integars 1-31|
|`HOUR`| Temporal Quantitative `dbl`| The hour within the day of the crime | Integars 0-23|
|`MINUTE`| Temporal Quantitative `dbl`| The minute within the hour of the crime | Integars 0-60 |
|`HUNDRED_BLOCK`| Categorical `str`| The block (specific location) on which the crime occurred) | 11275 unique values |
|`NEIGHBOURHOOD`| Categorical `str`| The neighbourhood the crime occurred in |    Has 25 unique values. These include: `['Fairview''West End', 'Central Business District', 'Grandview-Woodland', 'Mount Pleasant', 'Strathcona', 'Sunset', 'Kensington-Cedar Cottage', 'Stanley Park', 'Shaughnessy', 'Marpole', 'West Point Grey', 'Hastings-Sunrise', 'Victoria-Fraserview', 'Kitsilano''Kerrisdale', 'Riley Park', 'Oakridge', 'Arbutus Ridge', 'Renfrew-Collingwood', 'Killarney', 'Dunbar-Southlands', 'South Cambie', 'Musqueam', NA]`|
|`X`| Quantitative `dbl`| GPS longitude of crime | Has 3 NA values, and has 0's which are also NA values. |
|`Y`| Quantitative `dbl`| GPS latitude of crime | Has 3 NA values, and has 0's which are also NA values. |

## Question: